### ISSUE TITLE
Epson XP432 - blank pages (or garbage along left paper edge)

### ISSUE THREAD

**Issue by Groberen** [apple/cups#5772]
Hi.

OS: Fedora 31 x64.
CUPS 2.2.12 (the version that is included in this version of Fedora)
Printer: Epson XP-432

Information about printer in Printer tab: (http://localhost:631/printers/XP-432-435-Series)
Description: XP-432-435-Series
Driver: Generic ESC/P Dot Matrix Printer Foomatic/epson (recommended) (color)
Connection: usb://EPSON/XP-432%20435%20Series?serial=573242503031353581&interface=1
Defaults: job-sheets=none, none media=iso_a4_210x297mm sides=one-sided

Problem description:
Any attempt to print, from any software (tested with Xed, Libre Office, Gimp, Inkscape - all with same result), the printer spit out a blank page, or there may be random characters scattered along left edge of the paper.

I have ruled out defect printer, because I've tested it on a windows computer and I get it to print normally.

I've also tested the printer with Linux Lite 4.8, but unfortunately didn't checked the version of CUPS.

I don't know if this is a bug or something I do wrong. I hope I got to the tight place to get help on this issue.

**Comment by zdohnal** [comment]
Hi!

> Hi.
> 
> OS: Fedora 31 x64.
> CUPS 2.2.12 (the version that is included in this version of Fedora)
> Printer: Epson XP-432
> 
> Information about printer in Printer tab: (http://localhost:631/printers/XP-432-435-Series)
> Description: XP-432-435-Series
> Driver: Generic ESC/P Dot Matrix Printer Foomatic/epson (recommended) (color)
> Connection: usb://EPSON/XP-432%20435%20Series?serial=573242503031353581&interface=1
> Defaults: job-sheets=none, none media=iso_a4_210x297mm sides=one-sided

Thank you for the info! The printer is supported by epson-inkjet-printer-escpr package in Fedora, would you mind telling me whether you installed the print queue by yourself (in OS 'Settings'/CUPS web interface/system-config-printer) or it was installed automatically? Do you have the package installed?

If you installed it manually, please install the mentioned package and choose the correct driver.

If the print queue was installed automatically, then the issue can be in PackageKit (I checked epson-inkjet-printer-escpr, it generates the correct rpm tags, so it should be installed by PackageKit during running udev-configure-printer) or in udev-configure-printer (either in calling PackageKit or creating a print queue) - both are not part of CUPS.

To sum it up - if you installed the queue manually, install it with the correct driver and test printing - if it will not work, please provide debug logs (see https://fedoraproject.org/wiki/How_to_debug_printing_problems#How_to_get_and_attach_logs_for_bug_report ) .

If the queue was automatically installed, it is not a CUPS, but system-config-printer/PackageKit issue
 - in that case please open a bugzilla at bugzilla.redhat.com on system-config-printer. We can follow discussion there. This CUPS issue should be closed by you in that case.

**Comment by michaelrsweet** [comment]
Sorry, we don't support foomatic drivers here...
